packages/daemon/package.json

```diff
@@ -52,7 +53,7 @@
     "clear": "midway-bin clean",
     "dev": "cross-env NODE_ENV=local midway-bin dev --port=6927 --ts",
     "debug": "cross-env NODE_ENV=local midway-bin debug --port=6927 --ts",
-    "cov": "midway-bin cov --ts",
+    "cov": "midway-bin cov --ts --nyc='-r lcov -r text-summary'",
```

Comment:
Why do we use text-summary, it causes us to lose some information. See https://istanbul.js.org/docs/advanced/alternative-reporters/#text-summary.

Reply:
`-r lcov -r text-summary` means nyc will generate file lcov.info on disk and text-summary on console.